Packages/com.unity.inputsystem/Documentation~/Pen.md

```diff
@@ -1,6 +1,6 @@
 # Pen, tablet, and stylus support
 
-Pen support comprises both tablets on desktops, such as the various tablets produced by Wacom, and styluses on mobile devices, such as the stylus on the Samsung Note, the Apple Pencil on iOS, or the Surface Pen on the Microsoft Surface line of notebooks.
+Pen support comprises both desktop tablets (such as the various tablets produced by Wacom), and styluses on mobile devices (such as the stylus on the Samsung Note, the Apple Pencil on iOS, or the Surface Pen on the Microsoft Surface line of notebooks).
```

Comment:
The edit changed the meaning here: "tablets on desktops" means tablets connected to a desktop computer (the Wacom Intuos is one example: https://www.wacom.com/en-us/products/pen-tablets/wacom-intuos); "desktop tablets" is something else entirely (I *think* the Microsoft Surface technically qualifies as a desktop tablet).